js-exercises/drop-elements/dropElements.js

```diff
@@ -0,0 +1,11 @@
+function dropElements(elements, predicate) {
+  const remainingElements = [];
+  for (const el of elements) {
+    if (predicate(el)) {
+      remainingElements.push(el);
+    }
+  }
+  return remainingElements;
```

Comment:
Consider using the `filter` method in ES6. It would become as simple as:

`return elements.filter(predicate)`